src/main/java/com/academy/terai/Controller/AccountResource.java

```diff
@@ -0,0 +1,54 @@
+package com.academy.terai.Controller;
+
+import com.academy.terai.Model.Account;
+import com.academy.terai.Repository.AccountRepository;
+import com.academy.terai.service.AccountService;
+import javassist.NotFoundException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.management.openmbean.KeyAlreadyExistsException;
+import java.util.List;
+
+@RestController
+@RequestMapping("/accounts")
+public class AccountResource {
+    private final AccountService accountService;
+    @Autowired
+    public AccountResource(AccountService accountService){
+        this.accountService = accountService;
+    }
+    @GetMapping(value = "/all")
+    public List<Account> getAllAccounts() {
+        return accountService.findAll();
+    }
+
+    @GetMapping(value = "/{email}")
+    public Account getAccountByEmail(@PathVariable("email") String email) throws NotFoundException {
+        return accountService.findByEmail(email);
+    }
+
+    @GetMapping(value = "/orderByReviewedApplications")
+    public List<Account> findAllByOrderByReviewedApplicationsDesc() {
+        return accountService.findAllByOrderByReviewedApplicationsDesc();
+    }
+
+    @PostMapping
+    ResponseEntity<HttpStatus> createAccount(@RequestBody Account account) throws KeyAlreadyExistsException, NotFoundException {
+        accountService.addAccount(account);
+        return new ResponseEntity("Account added successfully", HttpStatus.OK);
```

Comment:
No need to put that text in here - I, as API consumer, asked your API to create the account, you returned 200 - all good with me, you don't need to provide any more details. Also, it'd be even better to use: https://restfulapi.net/http-status-204-no-content/